prep-javascript-variables/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    let fullName = 'Steve Hairston';
+    console.log(fullName);
+    let numberOfPets = 1;
+    console.log(numberOfPets);
+    let likesVegetables = 'true';
```

Comment:
For numbers, booleans, and `null`, you should leave out the quotes, because otherwise JavaScript sees a string.